lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -603,18 +589,37 @@
 
     },  
 
+    abbreviateNumber: function (value) {
+      var newValue = value = Math.round(value);
```

Comment:
Explain that this triple assign is doing please.

If you could break this into a less compact more understandable code it'd be great!